src/osemgrep/cli_interactive/Interactive_subcommand.ml

```diff
@@ -114,6 +118,15 @@ let fg_line_num = A.(fg neutral_yellow)
 *)
 let should_refresh = ref false
 
+let event_queue :
```

Comment:
please define the type separately, and if possible avoid Polymorphic variants; just use regular constructors.